### ISSUE TITLE
New Vaccine Valneva

### ISSUE THREAD

**Issue by testtrue** [Digitaler-Impfnachweis/certification-apis#247]
Hello @thinkberg ,

is it possible to generate certificates for the new vaccine Valneva? If so, what are the values for vp, mp and ma?

Greetings

**Comment by testtrue** [comment]
> #246

So our organization can take the values from that issue?

**Comment by kamianni** [comment]
> > #246
> 
> So our organization can take the values from that issue?

No, this is the thread where the discussion is taking place. However, there is no final statement yet.

**Comment by thinkberg** [comment]
It will be possible, as soon as the valuesets are updated (+ some time for the change necessary).

**Comment by kamianni** [comment]
Und wie steht es um die angepassten Impfstoffe von Biontech und Moderna? Wird es da auch Änderungen geben?

https://www.berliner-zeitung.de/news/neue-impfstoffe-derzeit-kein-digitales-impfzertifikat-moeglich-li.266826

**Comment by Ein-Tim** [comment]
@kamianni 

Please, could you open a new issue reg. this.

@thinkberg FYI

**Comment by kamianni** [comment]
Hello, are there any news regarding this topic?

**Comment by ChristianFischerJena** [comment]
Hello,
we are using the endpoint for vaccination centers only and using the "md" url-parameter to fill in the vaccine. I was not able to find a value set for this parameter. Could you please provide the correct parameter for valneva?

Kind regards
